pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -39,6 +39,9 @@
 from pyartcd.oc import get_release_image_info, get_release_image_pullspec, extract_release_binary, \
     extract_release_client_tools, get_release_image_info_from_pullspec, extract_baremetal_installer
 from pyartcd.runtime import Runtime, GroupRuntime
+from github import Github
+from github import GithubException
+from yaml.parser import ParserError
```

Comment:
Think the ruamel library has an exception that can be caught, let's not introduce the stdlib yaml parser here.

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/672/commits/995384aa85e26ab602932b4d690eec6b3f944a3c